public/apps/account/tenant-switch-panel.tsx

```diff
@@ -81,7 +81,7 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
 
         // @ts-ignore
         const currentRawTenantName = accountInfo.data.user_requested_tenant;
-        setCurrentTenant(currentRawTenantName, currentUserName);
+        setCurrentTenant(currentRawTenantName || '', currentUserName);
```

Comment:
Is this fixing any tenant? Global or Private?

Reply:
No it is not fixing any tenant. `setCurrentTenant` is method with signature `setCurrentTenant(string, string)` and `currentRawTenantName` is nullable string. Therefore handled while passing tenant to `setCurrentTenant` method.

Reply:
I see.